tasks/configure.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Place config file
+  template:
+    src: config.php.j2
+    dest: /srv/limesurvey/application/config/config.php
+    owner: www-data
+    group: www-data
+    mode: 0644
+
+- name: Place email config file
+  template:
+    src: email.php.j2
+    dest: /srv/limesurvey/application/config/email.php
+    owner: www-data
+    group: www-data
+    mode: 0644
+
+- name: Get db id of ldap plugin
+  shell: "mysql {{ limesurvey_mysql_database_name }} -Ns -e 'Select `id` from lime_plugins where `name`=\"AuthLDAP\";'"
+  register: limesurvey_ldap_plugin_id
+  when: limesurvey_use_ldap
+  changed_when: False
+
+- name: Update Ldap Plugin settings
+  shell: "mysql {{ limesurvey_mysql_database_name }} -Ns -e 'Update lime_plugin_settings set `value`=\"\\\"{{ item.value }}\\\"\" where `plugin_id`={{ limesurvey_ldap_plugin_id.stdout_lines[0] }} and `key`=\"{{ item.key }}\"; Select ROW_COUNT();'"
```

Comment:
I think you're missing `when: limesurvey_use_ldap` here

Reply:
done